client/src/components/Form.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import FormBasics from "./FormBasics";
+import FormAwards from "./FormAwards";
+import FormPublications from "./FormPublications";
+import FormLanguages from "./FormLanguages";
+import FormReferences from "./FormReferences";
+import FormWork from "./FormWork";
+import FormVolunteer from "./FormVolunteer";
+import FormEducation from "./FormEducation";
+import FormSkills from "./FormSkills";
+import FormInterests from "./FormInterests";
+
+export default function Form() {
+  return (
+    <section>
+      <form>
+        <FormBasics />
+        <FormWork />
+        <FormVolunteer />
+        <FormEducation />
+        <FormAwards />
+        <FormPublications />
+        <FormSkills />
+        <FormLanguages />
+        <FormInterests />
+        <FormReferences />
+        <div>
```

Comment:
I wasn't sure how I was going to build the sections of the submit form but this seems like it might work. Thanks.